ios/FluentUI/Drawer/DrawerController.swift

```diff
@@ -943,3 +970,13 @@ extension DrawerController: UIGestureRecognizerDelegate {
         return gestureRecognizer == resizingGestureRecognizer && otherGestureRecognizer == contentScrollView?.panGestureRecognizer
     }
 }
+
+// MARK: - DrawerController: DrawerPresentationDelegate
+
+extension DrawerController: DrawerPresentationDelegate {
+    func backgroundViewTapped(_ presentationControler: DrawerPresentationController) {
+        if delegate?.shouldDismissDrawer?(self) != false {
+            presentingViewController?.dismiss(animated: false)
```

Comment:
Why is it not animating the drawer dismissal?

Reply:
Thanks for pointing out, reverted this unintended change.